FileHeader.py

```diff
@@ -209,18 +209,36 @@ def get_strftime():
     return format
 
 
-def get_user():
-    '''Get user'''
+def get_author():
+    '''Get author'''
 
-    user = getpass.getuser()
-    output, error = getOutputError(
-        'cd {0} && git status'.format(get_dir_path()))
+    author = getpass.getuser()
+
+    prefix = 'cd %s && git ' % get_dir_path()
+
+    output, error = getOutputError(prefix + 'status')
 
     if not error:
-        output, error = getOutputError('git config --get user.name')
+        output, error = getOutputError(prefix + 'config --get user.name')
         if not error and output:
-            user = output
-    return user
+            author = output
+    return author
+
+
+def get_email():
```

Comment:
The structures of functions `get_author()` and `get_email()` are similar. Could you please refactor it?